pkgs/development/python-modules/hacking/default.nix

```diff
@@ -1,13 +1,13 @@
-{ lib
-, buildPythonPackage
-, fetchPypi
-, pbr
-, flake8
-, stestr
-, ddt
-, testscenarios
+{
+  lib,
+  buildPythonPackage,
+  fetchPypi,
+  pbr,
+  flake8,
+  stestr,
+  ddt,
+  testscenarios,
```

Comment:
Please revert the formatting changes. Alejandra does some weird stuff which is completely incompatible with nixfmt and nixpkgs-fmt.

Reply:
will do